src/dune_rules/setup.defaults.ml

```diff
@@ -1,11 +1,15 @@
 open! Dune_engine
 
-let library_path = None
-
-let library_destdir = None
-
-let mandir = None
-
-let docdir = None
-
-let etcdir = None
+let library_path = []
+
+let roots =
+  Install.Section.Paths.Roots.
```

Comment:
`Install.Section.Paths.Roots.` inside the record for consistency. Or a type annotation on the `roots` value.